### PR TITLE
Add GitHub Action to run MWC unit tests

### PR DESCRIPTION
**Pull request by aomarks**
For now only runs Chrome headless and Firefox headless. In a follow up, we can add Sauce integration.

### CODE REVIEW

File: .github/workflows/unit-tests.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-node@v1
+      with:
+          node-version: 10
```
Comment (azakus) [inline .github/workflows/unit-tests.yml]: Can we make this the 12 LTS, or "latest lts"?

Comment (aomarks) [reply .github/workflows/unit-tests.yml]: Pending feature request at https://github.com/actions/setup-node/issues/26 -- for now I just set 12.

**Comment by aomarks** [comment]
Also made Chrome/Firefox run concurrently instead of in separate steps.

**Comment by aomarks** [comment]
> Also made Chrome/Firefox run concurrently instead of in separate steps.

Which dropped time from 3m2s to 1m57s

File: .github/workflows/unit-tests.yml (hunk 1 above)
Comment (azakus) [reply .github/workflows/unit-tests.yml]: SGTM

**Pull request merged**